lib/streamlit/runtime/state/query_params_proxy.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
```

Comment:
I initially added this to `session_state_proxy.py` but then I felt like it should just be its own class based on how `query_params.py` exists. Open to moving this and having a different structure.